lib/reply.js

```diff
@@ -76,6 +84,11 @@ Reply.prototype.code = function (code) {
   return this
 }
 
+Reply.prototype.serializer = function (fn) {
+  this._serializer = fn
+  return this
+}
```

Comment:
should we allow to set a custom content-type straight from this function? it seems a must-have.